docs/Makefile

```diff
@@ -77,17 +87,25 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/Invenio.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/cookiecutterproject_name.qhcp"
```

Comment:
A cookiecutter leftover?  I would be good to `s/cookiecutterproject_name/invenio/g` here and in several other places.